ci-operator/config/openshift/cluster-monitoring-operator/openshift-cluster-monitoring-operator-master.yaml

```diff
@@ -45,6 +45,10 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9
   to: cluster-monitoring-operator
+- dockerfile_literal: |
```

Comment:
Using a literal dockerfile to publish an image to an imagestream seems pretty hacky. Do you think we could achieve the same result with promotion? https://docs.ci.openshift.org/docs/architecture/ci-operator/#publishing-to-an-integration-stream

Reply:
Actually that part is already merged: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-monitoring-operator/openshift-cluster-monitoring-operator-master.yaml#L48-L49

I wonder why GitHub is not warning about merge conflict :shrug: I'll rebase this PR.

---

It may potentially work with promotion. It is important though to create this config in separate PR as it needs to be on `master` branch to actually be used in e2e tests and rehearsal jobs.

Reply:
:exploding_head:

Looks good from my side then, let's just make sure to follow up on promotion to see if it's not a better fit for what we are trying to achieve.